pkg/test/utils_test.go

```diff
@@ -222,3 +199,72 @@ func find(s string, r []model.FormSubmission) bool {
 
 	return found
 }
+
+// set the environment variables for a test database
+// in the tests we are creating a unique database for testing
+// and removing it after running all the tests
+func setTestDatabase() {
+
+	// save back the MONGODB_URL env variable that it may be used for production
+	// this is sketchy and should be consider in the refactoring. We have the db package and web package getting MONGODB_URL everywhere
+	envmongodburl = os.Getenv("MONGODB_URL")
+
+	// use TEST_MONGODB_URL to create the connection to a new test database
+	mongodburl := os.Getenv("TEST_MONGODB_URL")
```

Comment:
But also there is the issue that in many parts of the code in the db and web package we are getting MONGODB_URL and connecting from there. It would be nice that we only get the MONGODB_URL at an app level.